Interfaces/DBView.xib

```diff
@@ -4124,6 +4125,9 @@ Gw
                             <segment toolTip="Forward" image="NSGoRightTemplate" width="27" enabled="NO" tag="1"/>
                         </segments>
                     </segmentedCell>
+                    <connections>
```

Comment:
I wonder if I f**ed up something or what happened... thanks James!

Reply:
Don't know .. but debugging I noticed `spHistoryControllerInstance` was always nil. Ended up diffing everything between 3010 and 3013:

![Screenshot 2021-01-13 at 10 08 36 PM](https://user-images.githubusercontent.com/1576190/104462831-ffaaef00-55eb-11eb-84a7-f1db0235cfa8.png)